### PR TITLE
Rename main view to home view

### PR DESCRIPTION
**Pull request by andidev**
the home view state is named "home" while the files are named main.
This fix changes all related code to home from main for consistency.

This also fixes a bug in cleanup.js file

### CODE REVIEW

File: app/cleanup.js [hunk 1]
```diff
@@ -56,5 +56,34 @@ function cleanupOldFiles(generator) {
         generator.removefile('profile_prod.gradle');
         generator.removefile('sonar.gradle');
         generator.removefile('yeoman.gradle');
+
+
+        generator.log(webappDir + 'scripts/app/main/main.controller.js');
```
Comment (deepu105) [inline app/cleanup.js]: is this intentional?

**Comment by pascalgrimaud** [comment]
Build failed :
```
  JHipster generator service
    creates service without interface
  ✓ creates service file
  ✓ doesnt create interface
    creates service with interface
  ✓ creates service file
  97 passing (11s)
  3 failing
  1) JHipster generator grunt default configuration creates expected default files:
      AssertionError: src/main/webapp/scripts/app/main/main.html, no such file or directory
      + expected - actual
```

There is so much refactor PR, and I'm happy to see my npm tests securing our changes

File: app/cleanup.js (hunk 1 above)
Comment (andidev) [inline app/cleanup.js]: Ouch though I removed that. I'll fix it later today.

**Comment by andidev** [comment]
@pascalgrimaud
Yea they are really helpfu! I'll have a look at it later today.

**Comment by andidev** [comment]
So I think this is ready to be merged now.

**Pull request merged**